### PR TITLE
Pull requesting f

### PR DESCRIPTION
**Pull request by Dincz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# CR-Backend
\ No newline at end of file
+started on 08-03-2023
```
Comment (noobj-nimverge) [inline README.md]: please format

File: controllers/contactController.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+const asyncHandler = require("express-async-handler");
+const Contact = require("../models/contactModel");
+const { constants } = require("../constants");
+
+// desc Get all contacts
+// route GET/api/contacts
+// access private
+const getContacts = asyncHandler(async (req, res) => {
+    const contacts = await Contact.find({ userId: req.user.id });
+    res.status(200).json(contacts);
```
Comment (noobj-nimverge) [inline controllers/contactController.js]: please read from constants instead of using hardcoded values?

File: controllers/userController.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    const user = await User.findOne({ email });
+    // compare password with hashed password
+    if (user && (await bcrypt.compare(password, user.password))) {
+        const accesstoken = jwt.sign({
+            user: {
+                username: user.username,
+                email: user.email,
+                id: user.id,
+            },
+        }, process.env.ACCESS_TOKEN_SECRET, { expiresIn: "15m" });
```
Comment (noobj-nimverge) [inline controllers/userController.js]: please dont use any hardcoded values

File: README.md
Comment (Lakshmi-kande) [inline README.md:1]: Please add space after '#' symbol

**Pull request merged**